packages/format-library/package.json

```diff
@@ -39,7 +39,8 @@
 		"@wordpress/icons": "*",
 		"@wordpress/private-apis": "*",
 		"@wordpress/rich-text": "*",
-		"@wordpress/url": "*"
+		"@wordpress/url": "*",
```

Comment:
I didn't see this dependency used.

Reply:
I can see that this dependancy already exists

https://github.com/WordPress/gutenberg/blob/2eec57974bb7df37f2d6b0c1583bd954f2126a2b/packages/format-library/package.json#L42C3-L42C24